.github/workflows/test-engine.yaml

```
@@ -0,0 +1,22 @@
+name: Go
+
+on:
+  push:
+  paths:
+    - 'engine/**'
```

```suggestion
    paths:
      - 'engine/**'
```
The syntax seems invalid